### PR TITLE
Add support for python3.5 and django1.9

### PR DESCRIPTION
**Pull request by ticosax**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
+sudo: false
 language: python
+python:
+  - "3.5"
```
Comment (pcraciunoiu) [inline .travis.yml]: Does this make it test ONLY python 3.5?

Comment (ticosax) [inline .travis.yml]: no, tox is still the one responsible to run tests in each version.
This change instruct travis-ci to install locally python3.5 which is not provided by default.

**Comment by crearc** [comment]
Also emits some Django 1.10 warnings:
```
/usr/local/lib/python2.7/dist-packages/django_ses/urls.py:7: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got dashboard). Pass the callable instead.
  url(r'^$', 'dashboard', name='django_ses_stats'),

/usr/local/lib/python2.7/dist-packages/django_ses/urls.py:7: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^$', 'dashboard', name='django_ses_stats'),
```

**Comment by pcraciunoiu** [comment]
That sounds like an easy fix. If you open a PR (especially one that tries to be backwards compatible), I am happy to merge it in. @crearc

**Comment by ticosax** [comment]
Any update on this one ?

**Comment by pcraciunoiu** [comment]
@ticosax Is the travis change making it test only python 3? That doesn't seem desirable.

The rest looks good at a glance. I haven't tested it.

**Comment by ticosax** [comment]
> @ticosax Is the travis change making it test only python 3?

Not at all, tox is still running the tests against all python versions.
specifying `python3.5` in travis.yaml makes the python3.5 interpreter available among others.

you can check in detail by yourself:
https://travis-ci.org/django-ses/django-ses/builds/126097128

**Pull request merged**